e/error-prone-annotations/error-prone-annotations_ubi_9.3.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : error-prone
+# Version       : 2.26.1
+# Source repo   : https://github.com/google/error-prone.git
+# Tested on     : UBI 9.3
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Pratibh Goshi<pratibh.goshi@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+set -e
+
+PACKAGE_NAME=error-prone/annotations
```

Comment:
Match the package name to the repo name. After the clone you can cd to the annotations directory.